library/std/src/sys/sync/once/mod.rs

```diff
@@ -21,7 +22,7 @@ cfg_if::cfg_if! {
         mod futex;
         pub use futex::{Once, OnceState};
     } else if #[cfg(any(
-        windows,
+        all(target_os = "windows", target_vendor = "win7"),
```

Comment:
This is using the `cfg_if!` macro so it can be kept as just `windows`. The first `if` branch will take precedence.